src/main/java/hazell/Command.java

```diff
@@ -0,0 +1,76 @@
+package hazell;
+
+import hazell.exceptions.KwargNotFound;
+import java.util.*;
+
+public class Command {
+    /**
+     * Represents userinput from user, parsed into arguments (args) and keyword arguments (kwargs).
+     *
+     * PARSING
+     * Format: [ARGS]... [/OPTION VALUE]...
+     * Options are specified by a slash (/) followed by a word. All non-option words that come after it
+     * are considered to be the value corresponding to that option.
+     *
+     * LIMITATIONS
+     * - Doesn't allow for multiple repeated options
+     */
+
+    private List<String> args;
+    private Map<String, String> kwargs;
+
+    private Command(List<String> args, Map<String, String> kwargs) {
+        this.args = args;
+        this.kwargs = kwargs;
+    }
+
+    /**
+     * Factory method for creating instances of the Command object.
+     * @param str Userinput
+     * @return A Command object
+     */
+    public static Command parse(String str) {
+        String[] splitBySlash = str.split("(?<=\\s)/");
+        List<String> args = new ArrayList<>(List.of(splitBySlash[0].split(" ")));
+
+        Map<String, String> kwargs = new HashMap<>();
+        for (int i = 0; i < splitBySlash.length - 1; i++) {
+            String[] kwargWords = splitBySlash[i + 1].split(" ", 2);
+            kwargs.put(kwargWords[0], kwargWords[1]);
+        }
+
+        return new Command(args, kwargs);
+    }
+
+    /**
+     * Checks whether the first arg matches `word`.
+     * @param word The first word in the command.
```

Comment:
Comment should have empty space between description of the method and the parameters.